core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporter.java

```diff
@@ -101,7 +59,7 @@ public static RequestContextExporterBuilder builder() {
         return new RequestContextExporterBuilder();
     }
 
-    private final ImmutableSet<BuiltInProperty> builtInPropertySet;
+    private final ImmutableSet<ExportEntry<BuiltInProperty>> builtInPropertySet;
```

Comment:
What do you think about using an array like other `ExportEntry`s for better export performance?

Reply:
Sure, changed `ImmutableSet` to an array.